liangBarskyAlgo/liangBarskyAlgo.c

```diff
@@ -0,0 +1,97 @@
+// Header files included
+#include<stdio.h>
+#include<conio.h>
```

Comment:
``conio.h`` is not a part of standard C/C++ library and is not supported by non DOS based systems. So remove this one.